e2e/typescript/tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "allowJs": true,
     "checkJs": true,
     "module": "CommonJS",
+    "lib": ["ES2019", "DOM", "DOM.Iterable"],
```

Comment:
To use `Object.fromEntries`. It seems that we just need to enable `ES2019` everywhere. I will do it in a separate PR.